src/styles/_variables.scss

```diff
@@ -25,7 +25,11 @@ $c-light-grey: #6E7686;
     --f-primary: "loos-normal", sans-serif;
     --f-mono: "DM Mono", monospace;
 
+    --page-max-width: 70rem;
     --page-padding: 1.5rem;
+    --top-bar-height: 3rem;
```

Comment:
You should let the scss file from the topbar component manage his own height.
Make a global variable for a top-bar-height wich will be use only in the top-bar component has no sence. Or maybe you think to something else ?

Reply:
I need it for positioning elements when position is fixed or absolute. Like in Page.scss I use it to position &__wallet-connector:

```
@import "../../styles/index";

.page {
    position: relative;
    margin: 0 auto;
    max-width: var(--page-max-width);

    &__content {
        position: relative;
        padding: var(--page-padding);
    }

    &__wallet-connector {
        position: absolute;
        top: calc(var(--top-bar-height) + var(--page-padding));
        left: var(--page-padding);
        width: calc(100% - var(--page-padding) * 2);
        z-index: 1;

        @include for-size(tablet-portrait-up) {
            top: calc(var(--desktop-top-bar-height) + var(--page-padding));
        }
    }
}
```

Reply:
Why you don't add a relative position with a top: var(--page-padding),or even a margin ? 
This content will be on overlay on something else ?

Reply:
> Why you don't add a relative position instead of absolute with a top: var(--page-padding),or even a margin ?
> This content will be on overlay on something else ?

Yeah I tried but failed. Try you and you'll see why.

> Also some quick question, i noticed that react doesn't scope the className to the component, i guess it is why you choose to use BEM convention. But why react doens't scope the scss file to the component which is the only who import the file ? Shouldn't we use className={styles.myClassName} to scope correctly the class and take the power of react instead of to manage that ourself with BEM ?

Give me an example and I might understand better what you mean.

Reply:
```
&__wallet-connector {
        margin-top: var(--page-padding);
        margin-left: var(--page-padding);
        width: calc(100% - var(--page-padding) * 2);
        // @include for-size(tablet-portrait-up) {
        //     top: calc(var(--desktop-top-bar-height) + var(--page-padding));
        // }
    }
```

Try this code, work on all devices. If this wallet buttons will never be in overlay, they shouldn't use absolute or z-index css. 
You can also move the height of the top-bar into his own css as it is not the responsability of the root variable to set the height of the component.

Reply:
But it needs to be absolute since it needs to fall over the content.